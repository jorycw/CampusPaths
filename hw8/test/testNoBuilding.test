# buildings not in Campus Paths
r
KNE
abc
r
abc
MGH
r
abc
cba
q