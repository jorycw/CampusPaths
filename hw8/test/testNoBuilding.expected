Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # buildings not in Campus Paths
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: r

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: r

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: r
Unknown building: r

Enter an option ('m' to see the menu):